methods/dig/index.js

```diff
@@ -0,0 +1,37 @@
+/**
+ * Retrieves the value object corresponding to
+ * each `index` objects repeatedly.
+ *
+ * @param {Array} array
+ * @param {...*} indices
+ *
+ * @return {Object|null}
+ *
+ * @example
+ * dig([[1, [2, 3]]], 0, 1, 1); // => 2
+ *
+ * @example
+ * rbjs([[1, [2, 3]]]).dig(0, 1, 1); // => 2
+ */
+export default function dig(array, ...indices) {
+    if (indices.length === 0) {
+        throw new TypeError('Invalid arguments');
+    }
+    let element;
+    while (indices.length > 0) {
+        let index = indices.shift();
+        if (index === null || index === undefined
+            || (['number', 'string'].some(e => typeof index === e)) === false
```

Comment:
@aldnav This can be made more readable by using `array.includes` instead of `array.some`.

Also, comparing `index` to `null` and `undefined` is already redundant, since `typeof null` and `typeof undefined` is `'object'` and `'undefined'` respectively, which is already neither `'number'` nor `'string'`.

Reply:
Thanks @arnellebalane Updated.